front/src/applications/stdcm/views/StdcmRequestModal.tsx

```diff
@@ -48,71 +54,77 @@ export default function StdcmRequestModal(props) {
   // https://developer.mozilla.org/en-US/docs/Web/API/AbortController
   const controller = new AbortController();
 
-  const timetableId = osrdconf.timetableID;
+  const timetableId = osrdconf.timetableID?.toString();
```

Comment:
Also, you should not cast this a string, but you can update the legacyOpenApi to make it a number (I checked in editoast, it must be a number)
```suggestion
  const timetableId = useSelector(getTimetableID);
```

Reply:
done